candi.sh

```diff
@@ -224,32 +229,62 @@ verify_archive() {
 
     # Skip verifying archive, if CHECKSUM=ignore
     if [ "${CHECKSUM}" = "ignore" ]; then
-        cecho ${WARN} "Skipped checksum check for ${ARCHIVE_FILE}"
+        cecho ${WARN} "Ignore checksum check for ${ARCHIVE_FILE}"
```

Comment:
well, "ignore" comes from my side and "skip" is/was the naming from dorsal. Maybe we should use CHECKSUM="skip" and the original cecho

Reply:
The remaining 'ignore' found by 'git grep ignore' belong to BUILDCHAIN=ignore.